server/apps/call_test.go

```diff
@@ -0,0 +1,39 @@
+package apps
```

Comment:
nm this simplistic test - there were silly issues with JSON unmarshaling, solved by restructuring. The test is not useful, but somewhat illustrative.